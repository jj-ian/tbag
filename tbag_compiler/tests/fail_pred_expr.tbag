a {
        endgame;
}

func int test() {
        int a = 8;
}


