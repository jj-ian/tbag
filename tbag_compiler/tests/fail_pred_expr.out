Fatal error: exception Failure("undeclared identifier a")
